src/components/shared/Modal.css

```diff
@@ -1,19 +1,23 @@
 .modal-wrapper {
   position: fixed;
   display: flex;
+  flex-direction: column;
   align-items: center;
+  justify-content: flex-start;
   width: 100%;
   height: 100%;
   top: 0;
   left: 0;
+  padding-top: 30px;
   transition: z-index 200ms;
   z-index: 100;
 }
 
 .modal {
   top: 0;
-  margin: auto;
   width: 500px;
+  max-height: 40vh;
+  height: 100%;
```

Comment:
we don't need this property, because it is overriden by `max-height: 40vh` in all cases